models/reviewsModel.js

```diff
@@ -19,4 +19,22 @@ const fetchReviews = () => {
     });
 };
 
-module.exports = { fetchReviews };
+const fetchReviewsById = (reviewId) => {
+  const changeToNumber = +reviewId;
+  if (isNaN(changeToNumber)) {
```

Comment:
Instead of this, we should be using psql's errors, so allowing the query to take place, catching the error, then sending back a 400

Reply:
couldnt get that approach to work. When I would console log my error.code I would get 22P02 but for some reason it wasnt passing to my error handler.